germanium/test_cases/migrations.py

```diff
@@ -0,0 +1,42 @@
+from .default import GermaniumTestCase
+from django.apps import apps
+from django.db.migrations.executor import MigrationExecutor
+from django.db import connection
+
+
+class MigrationTestCase(GermaniumTestCase):
+
+    @property
+    def app(self):
+        return self.strip_module_suffix(apps.get_containing_app_config(type(self).__module__).name)
+
+    def strip_module_suffix(self, module):
+        return module.split('.')[-1]
+
+    migrate_from = None
+    migrate_to = None
+
+    def setUp(self):
+        super().setUp()
+        self.set_up()
+        assert self.migrate_from and self.migrate_to, \
+            "TestCase '{}' must define migrate_from and migrate_to properties".format(type(self).__name__)
+        self.migrate_from = [(self.app, self.migrate_from)]
+        self.migrate_to = [(self.app, self.migrate_to)]
```

Comment:
tohle je takové zvláštní ty měníš proměnnou kterou tam někdo nastavil

Reply:
podle mě to ani nebude fungovat, jelikož se to volá před každým testem a ty to vždy před ním přenastavíš, takže víc testů povede k rekurzivnímu přenastavování a budeš mít list v listu v listu...